includes/class-wc-google-gtag-js.php

```diff
@@ -226,7 +226,7 @@ public static function listing_click( $product, $position ) {
 
 		wc_enqueue_js(
 			"
-			$( '.products .post-" . esc_js( $product->get_id() ) . " a' ).on('click', function() {
+			$( '.product.post-" . esc_js( $product->get_id() ) . ' a , .product.post-' . esc_js( $product->get_id() ) . " button' ).on('click', function() {
```

Comment:
On the `Products (beta)` block page it doesn't look like any of this inline code is included when viewing the page source. Tested with TT1 and TT3.

Reply:
Hi @martynmjones  sorry I forgot that Products Beta works differently and It will be needed to add another separated snippet because it doesn't include hooks. 

I think we can for now merge this PR which solves Block Based themes and "All products" blocks and address the PRoducts Beta in a separated PR. This is because 8.1 was released already. 

I created the issue here--- I will go for it tomorrow 
https://github.com/woocommerce/woocommerce-google-analytics-integration/issues/299

WDYT?

Reply:
Sounds good @puntope. There were no other issues so this PR should be good to merge then 👍